dist/angular-ui-tree.js

```diff
@@ -971,6 +989,17 @@
                   'top': topElmPos + 'px'
                 });
 
+                if (scrollContainerElm) {
```

Comment:
same comment as above, please add a comment how scrollContainerElm related to the code in the if